### PR TITLE
Package command apt get

### PR DESCRIPTION
**Pull request by mpreisler**
#### Description:

Split package_command into package_install and package_remove. Added apt-get support. Simplified a bunch of templates.

There is more work to be done, ubuntus are basically copies of each other and they each have a ton of non templated package install/remove fixes.

This is somewhat related to the jinja2 work but doesn't depend on it.

### CODE REVIEW

File: shared/bash_remediation_functions/package_install.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  echo "Aborting."
+  exit 1
+fi
+
+if which dnf ; then
+  if ! rpm -q --quiet "$package"; then
+    dnf install -y "$package"
+  fi
+elif which yum ; then
+  if ! rpm -q --quiet "$package"; then
```
Comment (jan-cerny) [inline shared/bash_remediation_functions/package_install.sh]: I think it is possible to have `yum` on Debian.
https://packages.debian.org/jessie/yum
Will this work in that case?

Comment (mpreisler) [reply shared/bash_remediation_functions/package_install.sh]: it should still work

Comment (shawndwells) [reply shared/bash_remediation_functions/package_install.sh]: YUM would be on debian, but not the RPM command.

Comment (yuumasato) [reply shared/bash_remediation_functions/package_install.sh]: `rpm` is listed as dependency of `yum`, so both will be there even on debian.

**Comment by yuumasato** [comment]
Generated content looks good, Ack!

**Pull request merged**